### PR TITLE
Verify that configuration file has proper ACLs during initialization

### PR DESCRIPTION
**Pull request by Damenus**

### CODE REVIEW

File: wca/main.py [hunk 1]
```diff
@@ -33,6 +34,30 @@
 log = logging.getLogger('wca.main')
 
 
+def valid_config_file(config):
+
+    user_uid = os.stat(config).st_uid
+    if user_uid != os.getuid() and user_uid != 0:
+        log.error(
+            'Error: The config \'%s\' is not valid. User is not owner config or is not root.'
```
Comment (mmucek95) [inline wca/main.py]: Please change "User is not owner config or is not root" to "User is not owner of the config or is not root"

File: wca/main.py [hunk 2]
```diff
@@ -33,6 +34,30 @@
...
+            % config)
+        exit(1)
+
+    mode = stat.S_IMODE(os.stat(config).st_mode)
+    if mode != 700 and mode != 600:
+        log.error(
+            'Error: The config \'%s\' is not valid. It has not correct proper ACLs. '
```
Comment (mmucek95) [inline wca/main.py]: Please use either correct or proper. Not both. Maybe something like this: "It doesn't have correct ACLs. "

File: wca/main.py [hunk 3]
```diff
@@ -33,6 +34,30 @@
...
+            'Only user must be able to read and write.'
```
Comment (mmucek95) [inline wca/main.py]: s/must/should souns better IMHO

File: wca/main.py (hunk 1 above)
Comment (Damenus) [reply wca/main.py]: done

File: wca/main.py (hunk 3 above)
Comment (Damenus) [reply wca/main.py]: done

File: wca/main.py (hunk 2 above)
Comment (Damenus) [reply wca/main.py]: done

**Pull request merged**